cli/daemon/test_process/test_process.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"os"
+
+	"github.com/tarantool/tt/cli/daemon"
+	"github.com/tarantool/tt/cli/ttlog"
+)
+
+type TestWorker struct {
```

Comment:
Add a description for the public struct and methods. (here and bellow)

Reply:
Done.